pkgs/top-level/python-packages.nix

```diff
@@ -2452,6 +2452,24 @@ let
     };
   };
 
+  pytestpep257 = buildPythonPackage rec {
+     name = "pytest-pep257-${version}";
+     version = "0.0.1";
+
+     src = pkgs.fetchurl {
+       url = "https://pypi.python.org/packages/source/p/pytest-pep257/${name}.tar.gz";
+       sha256 = "003vdkxpx37n0kjqpwgj3314hwk2jfz3nz58db7xh68bf8xy75lk";
+     };
+
+     propagatedBuildInputs = with self ; [ pytest pep257 ];
+
+     meta = {
+       website = "https://github.com/anderslime/pytest-pep257";
+       description = "py.test plugin for pep257";
```

Comment:
`py.test` is the name of the testing framework this plugs in (`pythonPackages.pytest` in `nixpkgs`), and `PEP257` the name of the standard this plug-in checks code against. The description (I actually picked from the project homepage) only refers to the same elements the package name is built on.